pkg/debug/server.go

```diff
@@ -36,6 +37,10 @@ func (s *Server) WithPrefix(prefix string) {
 	s.Prefix = prefix
 }
 
+func (s *Server) WithHttpPort(port string) {
+	s.HttpPort = port
+}
```

Comment:
Could we refactor this a bit, please.
We don't use override or something like this in the runtime. So we can use 
```go
func NewServer(prefix, socketPath, httpPort string) *Server { }
```
signature and initialize it in the go-way